doc/source/user_guide/deployment_slurm.rst

```diff
@@ -0,0 +1,250 @@
+
+SLURM script file
+~~~~~~~~~~~~~~~~~
+
+In the SLURM script, you'll need to tell SLURM to allocate nodes for your Xorbits job. 
+In this example, we ask for 4 nodes, and on each node, we've set ``--cpus-per-task=24`` and ``--ntasks-per-node=1`` which means we need 24 CPUs per node. 
+Modify this setting according to your workload. Similarly, you can also specify the number of GPUs per node via ``--gpus=1``.
+You need to change ``--partition`` to select the partition in your site. You can also add other optional flags to your ``sbatch`` directives.
+
+.. code-block:: bash
+
+    #!/bin/bash
+    #SBATCH --job-name=xorbits
+    #SBATCH --nodes=4
+    #SBATCH --cpus-per-task=24
+    #SBATCH --ntasks-per-node=1
+    #SBATCH --partition=cpu24c
+    #SBATCH --time=00:30:00
+
+
+Loading your environment
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+You'll need to install Xorbits into a specific environment using ``conda`` or ``module``. 
+In the SLURM script, you should load modules or your own conda environment. 
+And on the compute nodes allocated, the environment will be switched to the one with Xorbits installed.
+In this case, we install Xorbits in a conda environment called ``df``.
+
+.. code-block:: bash
+
+    # Example: module load xorbits
+    # Example: source activate my-env
+
+    source activate df
+
+
+Obtain the nodes
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Next, we'll want to obtain the head node and the IP address.
+
+.. code-block:: bash
+
+    # Getting the node names
+    nodes=$(scontrol show hostnames "$SLURM_JOB_NODELIST")
+    nodes_array=($nodes)
+
+Now the ``nodes_array`` is the list of all the nodes allocated for this job.
+
+Choose the first node of the ``nodes_array`` as the head node. 
+The ``hostname --ip-address`` command will return the IP address of the specific node. 
+Get the IP address of the head node:
+
+.. code-block:: bash
+
+    head_node=${nodes_array[0]}
+    head_node_ip=$(srun --nodes=1 --ntasks=1 -w "$head_node" hostname --ip-address)
+
+    # if we detect a space character in the head node IP, we'll
+    # convert it to an ipv4 address. This step is optional.
+    if [[ "$head_node_ip" == *" "* ]]; then
+        IFS=' ' read -ra ADDR <<<"$head_node_ip"
+        if [[ ${#ADDR[0]} -gt 16 ]]; then
+            head_node_ip=${ADDR[1]}
+        else
+            head_node_ip=${ADDR[0]}
+        fi
+        echo "IPV6 address detected. We split the IPV4 address as $head_node_ip"
+    fi
+
+Starting the supervisor
+~~~~~~~~~~~~~~~~~~~~~~~
+
+After detecting the head node hostname and IP address, we'll want to run the supervisor on the head node. 
+We'll do this by using ``srun`` to start the supervisor on the head node. 
+``xorbits-supervisor`` is the command line tool to start the supervisor.
+You should specify the IP, port, the web port.
+Note that you should sleep a few seconds as the supervisor need some time to start. Otherwise, other worker nodes may not able to connect to the supervisor.
```

Comment:
```suggestion
Note that you should sleep a few seconds as the supervisor need some time to start. Otherwise, worker nodes may not be able to connect to the supervisor.
```

Reply:
This issue seems not be resolved yet. @luweizheng